### PR TITLE
Add Kibana Proxy settings per LOG-276

### PR DESCRIPTION
**Pull request by mburke5678**
https://jira.coreos.com/browse/LOG-276

### CODE REVIEW

**Comment by mburke5678** [comment]
@anpingli @jcantrill Please take a look. 
Do we need to define what a Kibana proxy is? (If so, what is a Kibana proxy?). Is it optional?

**Comment by jcantrill** [comment]
> @anpingli @jcantrill Please take a look.
> Do we need to define what a Kibana proxy is? (If so, what is a Kibana proxy?). Is it optional?

It is not optional.  I'm indifferent if we should define it.  It's basically the thing which controls access to Kibana defering auth to openshift.  This allows us to provide an auth mechanism without needing to modify Kibana.

File: modules/efk-logging-kibana-limits.adoc [hunk 1]
```diff
@@ -5,18 +5,20 @@
 [id="efk-logging-kibana-limits-{context}"]
 = Configure Kibana CPU and memory limits
 
-Each component specification allows for adjustments to both the CPU and memory limits. 
+Each component specification allows for adjustments to both the CPU and memory limits. You should not need to manually adjust these values as the Cluster Logging Operator sets values sufficient for your environment.
```
Comment (jcantrill) [inline modules/efk-logging-kibana-limits.adoc]: I would refrain from making this statement as the Operator yet is unable to make decisions about the deployment related to the characteristics of the cluster.  We set defaults based on experience from previous releases but admins may need to adjust them

File: modules/efk-logging-kibana-limits.adoc [hunk 2]
```diff
@@ -5,18 +5,20 @@
...
 
 .Prerequisite
 
-If needed, get the name of the Cluster Logging Custom Resource from the `openshift-logging` project:
-
+* If needed, get the name of the Cluster Logging Custom Resource from the `openshift-logging` project:
```
Comment (jcantrill) [inline modules/efk-logging-kibana-limits.adoc]: When https://github.com/openshift/cluster-logging-operator/pull/123 merges the resource will be named 'instance'

File: modules/efk-logging-kibana-limits.adoc [hunk 3]
```diff
@@ -5,18 +5,20 @@
...
++
 ----
 $ oc get ClusterLogging
 NAME       AGE
 instance   112m
 ----
 
+* Set cluster logging to the unmanaged state.
```
Comment (jcantrill) [inline modules/efk-logging-kibana-limits.adoc]: This is not needed in order to adjust the proxy settings.

File: modules/efk-logging-kibana-scaling.adoc [hunk 4]
```diff
@@ -39,16 +39,7 @@ spec:
...
-              cpu:
-              memory:
-            requests:
-              cpu:
-              memory:
 ---- 
 
 <1> Specify the number of Kibana nodes.
-<2> Specify the CPU and memory limits for each node.
+
```
Comment (jcantrill) [inline modules/efk-logging-kibana-scaling.adoc]: I don't expect the need to adjust the proxy but you could leave some note here, but it's not per node but for the proxy.

Comment (mburke5678) [reply modules/efk-logging-kibana-scaling.adoc]: @jcantrill I had removed this line from the file and created a new file for memory requests. 
https://github.com/openshift/openshift-docs/pull/14210/files#diff-54aee4af720c50efd3731f628998fb35R59

File: modules/efk-logging-kibana-limits.adoc (hunk 3 above)
Comment (mburke5678) [reply modules/efk-logging-kibana-limits.adoc]: @jcantrill What about for memory and CPU requests for Kibana (not the proxy)?

**Comment by mburke5678** [comment]
@anpingli PTAL

**Comment by mburke5678** [comment]
@anpingli Please take a look.

File: modules/efk-logging-kibana-limits.adoc (hunk 3 above)
Comment (anpingli) [reply modules/efk-logging-kibana-limits.adoc]: @mburke5678  The change is in Cluster Logging Custom Resource, the cluster logging must be  in Managed state. 

The other Part LGTM

**Pull request merged**